vr/common/tests/test_repos.py

```diff
@@ -21,53 +20,6 @@ def _git_config():
     run('git config --global user.name "Your Name"')
 
 
-@skipif_nohg
```

Comment:
This is a bit of a brutal way to "fix" a failing test. Removing the test effectively means dropping support for the tested feature, IMO. Can't instead just point the test to another public mercurial repository?

Reply:
Since bitbucket no longer supports mercurial I'm not sure what the alternative would be. Sourceforge? I'm inclined to go forward with the removal of the mercurial test, or configure the test runner so we stop running the mercurial related tests rather than just removing the test.

Reply:
At least, we can mark the test as an "expected failure" but leave it around, if we can't find a public Mercurial repo to peruse.

Reply:
@lbolla  I've updated the test and add the expect to faill, can we proceed forward?

Reply:
Sure thanks